### PR TITLE
add language selector

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review please

File: molo/core/cookiecutter/scaffold/{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/settings/base.py [hunk 1]
```diff
@@ -115,6 +117,23 @@
...
+LANGUAGES = global_settings.LANGUAGES + (
+    ('zu', _('Zulu')),
+    ('xh', _('Xhosa')),
+    ('st', _('Sotho')),
+    ('ve', _('Venda')),
+    ('tn', _('Tswana')),
+    ('ts', _('Tsonga')),
+    ('ss', _('Swati')),
+    ('nr', _('Ndebele')),
+    )
```
Comment (smn) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/settings/base.py]: :heart:

File: molo/core/cookiecutter/scaffold/{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/templates/404.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{% raw %}
```
Comment (smn) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/templates/404.html]: clever :)

**Comment by smn** [comment]
Nice work dude. :+1:

**Pull request merged**